_posts/2018-05-02-looking-at-rxjs-schedulers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: What are those schedulers in RxJS
```

Comment:
What are schedules in RxJS might be better?

Reply:
al vergeleken met die van staltz?https://staltz.com/primer-on-rxjs-schedulers.html

Reply:
Beide gelezen? Hij covered ook iets ivm schedulers maar beperkt. Ik leg uit welke er zijn en hoe ze gelinkt zijn aan welke async 'paradigm' in RxJS.